2005-01-14  GOTO Masanori  <gotom@debian.or.jp>

        * manual/string.texi (strcasestr): Fix example typo.

===================================================================
RCS file: /cvs/glibc/libc/manual/string.texi,v
retrieving revision 1.69
retrieving revision 1.70
diff -u -r1.69 -r1.70
--- libc/manual/string.texi	2004/07/14 17:10:28	1.69
+++ libc/manual/string.texi	2005/01/26 02:39:11	1.70
@@ -1781,9 +1781,9 @@
 
 For example,
 @smallexample
-strstr ("hello, world", "L")
+strcasestr ("hello, world", "L")
     @result{} "llo, world"
-strstr ("hello, World", "wo")
+strcasestr ("hello, World", "wo")
     @result{} "World"
 @end smallexample
 @end deftypefun
